geo/src/algorithm/affine_ops.rs

```diff
@@ -127,6 +183,10 @@ impl<T: CoordNum> AffineTransform<T> {
         )
     }
 
+    pub fn is_identity(&self) -> bool {
```

Comment:
This came in handy while trying to skip application in my code base.

Reply:
Could we get a brief example (or explanation as a doc comment)?